mtp_noms_ops/assets-src/stylesheets/views/_checks.scss

```diff
@@ -171,20 +164,34 @@ table.mtp-table--review {
 .mtp-check-cell__actions {
   display: flex;
   align-items: center;
+
+  .button, .button-secondary {
+    margin-bottom: 1rem;
+
+    @include media(mobile) {
+      width: auto;
+    }
+  }
 }
+
 .mtp-check-cell__list-status {
   margin-left: 1rem;
 }
 
-.mtp-check-cell__list-status--my-list {
+.mtp-check-cell__list-status--my-list, .mtp-check-cell__list-status--text {
   display: inline-block;
-  margin-bottom: 1.6rem;
+  margin-bottom: 0.6rem;
 }
 
 .mtp-check-cell__list-status--my-list::before {
-	content: "✓";
-	font-size: 1.5em;
-	color: #00703c;
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  margin-top: -5px;
+  vertical-align: middle;
+  content: '';
+  background: file-url('added-to-my-list.png') no-repeat;
+  background-image: none, file-url('added-to-my-list.svg');
```

Comment:
Why do we need to override the image value from the background shorthand property here?

Reply:
this is a way to make it work on older browsers. newer browsers that understand svg also understand multiple background images, whereas older ones won't understand the `background-image` directive and not use it showing just a png. you can't do `background: file-url('added-to-my-list.png'), file-url('added-to-my-list.svg') no-repeat;` because that includes both images

Reply:
Ah, now it makes sense, I missed that the first image was a png